superduperdb/backends/base/artifact.py

```diff
@@ -42,15 +42,15 @@ def url(self):
         pass
 
     @abstractmethod
-    def _delete_bytes(self, file_id: str):
+    def _delete_artifact(self, file_id: str):
```

Comment:
BASE64 only intended to be saved in documents. So this is actually always `bytes`.